src/python/pants/backend/python/rules/coverage.py

```diff
@@ -98,6 +108,13 @@ def register_options(cls, register):
             advanced=True,
             help="Path to write the Pytest Coverage report to. Must be relative to build root.",
         )
+        register(
+            "--config",
+            type=file_option,
+            default=None,
+            advanced=True,
+            help="Path to `.coveragerc` or alternative coverage config file",
```

Comment:
> "alternative coverage config file"

I'm not sure what this means... does it mean something like "or other file in the `.coveragerc` format"?

Reply:
That wording comes from the other subsystems. It's meant to say that it could for example be `pyproject.toml`. See all the options they have at https://coverage.readthedocs.io/en/coverage-5.2/config.html.

Reply:
copied from here: https://github.com/pantsbuild/pants/blob/master/src/python/pants/backend/python/lint/flake8/subsystem.py#L33